internal/pkg/template/env.go

```diff
@@ -44,7 +48,7 @@ var leastVersionForFeature = map[string]string{
 	NATFeatureName:                     "v1.3.0",
 	InternalALBFeatureName:             "v1.10.0",
 	AliasesFeatureName:                 "v1.4.0",
-	AppRunnerPrivateServiceFeatureName: "v1.23.0",
+	AppRunnerPrivateServiceFeatureName: "v1.13.0",
```

Comment:
can we add a comment to this variable to highlight if these versions are environment template versions or copilot versions